src/lib/parts.ts

```diff
@@ -93,7 +115,12 @@ export class AttributeCommitter {
   commit(): void {
     if (this.dirty) {
       this.dirty = false;
-      this.element.setAttribute(this.name, this._getValue() as string);
+      let value = this._getValue() as string;
+      if (typeof value === 'symbol') {
```

Comment:
moving the check for symbol into _getValue() might save a couple of bytes

Reply:
Done